src/lib.rs

```diff
@@ -66,6 +66,9 @@
 //!   atomic operations on platforms that don't support them.
 //!
 //! [`portable-atomic`]: https://crates.io/crates/portable-atomic
+//!
+//! - The `std` feature enables the use of the Rust standard library. Disable it for `no_std`
```

Comment:
```suggestion
//! - The `std` feature (enabled by default) enables the use of the Rust standard library. Disable it for `no_std`
```

Also, it might be better to swap the order of this and `portable-atomic` feature to match the order in Cargo.toml.